requirements.txt

```diff
@@ -1,58 +1,46 @@
-#
```

Comment:
This entire area needs to be cleaned up and double checked. @Madscientiste @SandroHc

Reply:
You're right. The `requirements.txt` shouldn't be changed directly as we're now using Pipenv for dependency management.

The correct way is to install the dependency via Pipenv, `pipenv install <dep>`, then regenerate this file via `pipenv freeze -r > requirements.txt`.